app/assets/config/manifest.js

```diff
@@ -0,0 +1,5 @@
+//= link_tree ../images
+//= link application.js
+//= link webchat.js
+//= link application.css
+//= link print.css
```

Comment:
This looks great nice job.

To finish you can remove this code https://github.com/alphagov/government-frontend/blob/f00bc70be9e48fddb0d120b86f694025f515d22e/config/initializers/assets.rb#L6-L15

Reply:
Updated as suggested, thanks.